setup.py

```diff
@@ -23,7 +24,7 @@ def read(*names, **kwargs):
         re.compile("^.. start-badges.*^.. end-badges", re.M | re.S).sub(
             "", read("README.rst")
         ),
-        re.sub(":[a-z]+:`~?(.*?)`", r"``\1``", read("CHANGELOG.rst")),
+        re.sub(":[a-z]+:`~?(.*?)`", r"``\1``", "https://dave-data.readthedocs.io/en/latest/changelog.html"),
     ),
```

Comment:
Are you sure that this will work. You are replacing a read-function with a string.

Reply:
The read function also returns a string, so this shouldn't be a problem

Reply:
```python
 long_description="{}\n{}".format(
        re.compile("^.. start-badges.*^.. end-badges", re.M | re.S).sub(
            "", read("README.rst")
        ),
        "CHANGELOG\n=========\n\n`Read the actual changelog at Readthedocs "
        "<https://dave-data.readthedocs.io/en/stable/changelog.html>`_",
    )
),
 ```